addon-test-support/scroll-into-view-mock.js

```diff
@@ -0,0 +1,32 @@
+import { find } from '@ember/test-helpers';
+
+export default function mockScrollIntoView() {
+  const elementsInvokedOn = [];
```

Comment:
Replace all instances of `const` with `let` in this file.

Reply:
is that so it supports older browsers?

Reply:
No, it's because these aren't constants so we shouldn't be marking them as `const`.